clang/bindings/python/clang/cindex.py

```diff
@@ -66,46 +66,77 @@
 
 import collections.abc
 import os
+import sys
```

Comment:
This is used for `sys.stdout.flush()` in this file, and leads to a type error. I don't know why this isn't already imported